modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/notice_of_disagreements/evidence_submissions_controller.rb

```diff
@@ -15,6 +16,16 @@ def show
 
           render json: serialized.serializable_hash
         end
+
+        def create
+          status, error = AppealsApi::FileValidator.new(params[:document]).call
+
+          if status == :ok
+            render json: { document: params[:document], uuid: params[:uuid] }
+          else
+            render json: error, status: 422
```

Comment:
Check out [the error docs on JSON:API](https://jsonapi.org/format/1.0/#errors). As noted:
> ... Error objects *MUST* be returned as an array keyed by errors in the top level of a JSON:API document. ...

```suggestion
            render json: { errors: [error] }, status: 422
```